### ISSUE TITLE
No support for $INCLUDE

### ISSUE THREAD

**Issue by larsks** [aredwood/zonefile-parser#19]
More a feature request than a bug.

I was hoping to use this module to parse some local zone files, but we're using the `$INCLUDE` directive and this causes `zonefile_parser.parse` to fail with:

```
Traceback (most recent call last):
  [...]
  File ".../site-packages/zonefile_parser/parser.py", line 38, in parse_record
    "value":parts[RecordEnum.RDATA]
IndexError: list index out of range
```

**Comment by aredwood** [comment]
could you send a few samples my way? so i could write a few tests in.

**Comment by larsks** [comment]
I mean, it's just `$INCLUDE path/to/file`, and then you parse records from `path/to/file`. I'll attach a sample shortly.

**Comment by larsks** [comment]
E.g. https://gist.github.com/larsks/10676a258909b8d00cbb7ad1ebefc413

**Comment by aredwood** [comment]
Hey, I'm taking a look now.

The issue here is that the package takes in a stream and parses it, meaning is has no context of where the file came from and is therefore unable to resolve relative file paths without additional context.

**Comment by aredwood** [comment]
Released in 0.1.17
